text/0117-typo-setting-api.md

```diff
@@ -0,0 +1,295 @@
+# Typo Tolerance Settings API
+
+## 1. Summary
+
+This specification describes the customizable options for the typo tolerance settings.
+
+## 2. Motivation
+
+Tolerance to typos is critical when building an enjoyable search experience. Meilisearch ranks typo-free matched documents above others based on the default settings that we believe are best for most users.
+
+Meilisearch can adapt to different use-cases by providing customization options for the typo tolerance feature.
+
+## 3. Functional Specification
+
+### 3.1. `typo` API resource definition
+
+| Field                                            | Type            | Required |
+|--------------------------------------------------|-----------------|----------|
+| [enabled](#311-enabled)                          | Boolean         | False    |
+| [disabledAttributes](#312-disabledattributes)    | Array of String | False    |
+| [disabledWords](#313-disabledwords)              | Array of String | False    |
+| [minWordSizeForTypos](#314-minwordsizefortypos)  | Object          | False    |
+
+#### 3.1.1. `enabled`
+
+- Type: Boolean
+- Required: False
+- Default: `true`
+
+Whether the typo tolerance feature is enabled.
+
+##### 3.1.1.1. Impacts on the `typo` ranking rule
```

Comment:
1. As long as the `typo` rule is part of the `rankingRules` array, it will be applied regardless of if typo tolerance is enabled or not, correct? 
2. Not sure this even makes sense, but how is the `typo` ranking rule applied if you enable typo tolerance? Will it work the same way but match fewer documents?

Reply:
The presence of `typo` in the `rankingRules` does not enable or disable the typo tolerance feature.

If `enabled` is false (the typo-tolerance feature is not used to match documents), the `typo` ranking rules have no impact.
if `enabled` is true (the typo-tolerance feature is used to match documents containing typos), the `typo` ranking rules will rank documents according to the number of typos found (`0` the better). If the `typo` rule is not defined in the `rankingRules` settings, documents containing typos will be matched but the `typo` ranking rule will not be used to sort them.

Does it help? Don't hesitate to tell me if I can be more precise. 😇